src/jdk.compiler/share/classes/com/sun/tools/javac/code/Preview.java

```diff
@@ -46,6 +47,7 @@
 import java.util.Map;
 import java.util.Set;
 
+import static com.sun.tools.javac.code.Flags.PREVIEW_API;
```

Comment:
```suggestion
```
Redundant import (sorry i should have checked before i sent you updates to this area)

Reply:
Merged